commands/goto_line.go

```diff
@@ -84,6 +84,9 @@ func (g *GotoLine) Exec(on interface{}) (executed, consume bool) {
 	if line >= g.editor.Controller().LineCount() {
 		g.err = fmt.Sprintf("Line %d is past the end of the file", line)
 		return true, true
+	} else if line == -1 {
```

Comment:
The else here is unnecessary, since the previous `if` statement returns.  Can we change it to just an `if`?